app/assets/stylesheets/_search_results.scss

```diff
@@ -71,7 +71,8 @@
     width: 100%;
   }
 
-  &__button {
+  &__button,
+  .sign__button {
```

Comment:
The .sign__button class is another one that looks like it isn't used anywhere? You should be able to remove that as well now and just have these styles assigned to the &__button element.